### PR TITLE
[App Service] Change single quotes to back ticks

### PR DESCRIPTION
**Pull request by ammanifold**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by ammanifold** [comment]
Fixes #7154

**Comment by qwordy** [comment]
Looks good to me. But remember to fix checks errors.

**Comment by ammanifold** [comment]
Thanks @qwordy, All checks have passed

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -7,6 +7,10 @@ Release History
 
 * Add support for importing/exporting feature flags
 
+**AppService**
+
+* Fix issue #7154: webapp:Single quotes instead of back ticks
```
Comment (panchagnula) [inline src/azure-cli/HISTORY.rst]: can you say updating documentation for command <> to use quoted instead of back ticks?

Comment (ammanifold) [reply src/azure-cli/HISTORY.rst]: It's the other way around but yes :)

**Pull request merged**